docs/server-dependencies.md

```diff
@@ -0,0 +1,149 @@
+---
+id: server-dependencies
+title: Temporal server dependencies
+sidebar_label: Dependencies
+---
+
+## Overview
+
+This page details some of the requirements and dependencies needed to build and run an instance of Temporal.
+
+All versions of Temporal require that [Go v1.15+](https://golang.org/dl/) is installed in the environment.
+
+## Dependency matrix
+
+The following table shows which versions of dependencies are supported with a specific version of Temporal.
+
+Note that the only required dependency is a database, and there are multiple types of databases that are supported. Search is an optional feature and currently only ElasticSearch is supported. Event streaming software as a dependency is only required when ElasticSearch is being used or when Temporal is deployed across multiple data centers, however in future releases of Temporal, third party event streaming software will likely cease to be needed as dependency for both. Temporal emits metrics by default in a format that is supported by Prometheus. Monitoring and observing those metrics is optional. Any software that can pull metrics that supports the same format could be used, but we only ensure it works with Prometheus and Grafana versions.
+
+Legend of tested and supported dependency versions:
+
+- 🟩 &nbsp;**Cassandra v3.11**
+- 🟪 &nbsp;**MySQL v5.7**
+- 🟧 &nbsp;**PostgreSQL v9.6**
+- 🟥 &nbsp;**ElasticSearch v6.8**
+- 🟦 &nbsp;**Prometheus >= v2.0**
+- 🟨 &nbsp;**Grafana >= v2.5**
+- 🟫 &nbsp;**Kafka v2.1.1 & Zookeeper v3.4.6**
+
+
+| Version | Go packages | Databases | Search | Event streaming | Monitoring | Observation |
+|---------|-------------|-----------|--------|-----------------|------------|-------------|
+| [1.3.2](https://github.com/temporalio/temporal/tree/v1.3.2) | [go.mod](https://github.com/temporalio/temporal/blob/v1.3.2/go.mod) | 🟩 🟪 🟧 | 🟥 | 🟫 | 🟦 | 🟨 |
+| [1.3.1](https://github.com/temporalio/temporal/tree/v1.3.1) | [go.mod](https://github.com/temporalio/temporal/blob/v1.3.1/go.mod) | 🟩 🟪 🟧 | 🟥 | 🟫 | 🟦 | 🟨 |
+| [1.3.0](https://github.com/temporalio/temporal/tree/v1.3.0) | [go.mod](https://github.com/temporalio/temporal/blob/v1.3.0/go.mod) | 🟩 🟪 🟧 | 🟥 | 🟫 | 🟦 | 🟨 |
+| [1.2.2](https://github.com/temporalio/temporal/tree/v1.2.2) | [go.mod](https://github.com/temporalio/temporal/blob/v1.2.2/go.mod) | 🟩 🟪 🟧 | 🟥 | 🟫 | 🟦 | 🟨 |
+| [1.2.1](https://github.com/temporalio/temporal/tree/v1.2.1) | [go.mod](https://github.com/temporalio/temporal/blob/v1.2.1/go.mod) | 🟩 🟪 🟧 | 🟥 | 🟫 | 🟦 | 🟨          |
+| [1.1.1](https://github.com/temporalio/temporal/tree/v1.1.1) | [go.mod](https://github.com/temporalio/temporal/blob/v1.1.1/go.mod) | 🟩 🟪 | 🟥 | 🟫 | 🟦 | 🟨 |
+| [1.1.0](https://github.com/temporalio/temporal/tree/v1.1.0)   | [go.mod](https://github.com/temporalio/temporal/blob/v1.1.0/go.mod) | 🟩 🟪 | 🟥 | 🟫 | 🟦 | 🟨 |
+| [1.0.0](https://github.com/temporalio/temporal/tree/v1.0.0) | [go.mod](https://github.com/temporalio/temporal/blob/v1.0.0/go.mod) | 🟩 🟪 | 🟥 | 🟫 | 🟦 | 🟨 |
```

Comment:
just a suggestion to reorg info, take or leave as you wish

```suggestion
Temporal offers official support for and is tested against dependencies with these exact versions:

- Databases (Must have one): Note that the only required dependency is a database, and there are multiple types of databases that are supported. 
	- **Cassandra v3.11**
	- **MySQL v5.7**
	- **PostgreSQL v9.6** (supported since Temporal v1.2.1)
- Search (Optional): Search is an optional feature and currently only ElasticSearch is supported. 
	- **ElasticSearch v6.8**
- Monitoring & Observability (Optional): Temporal emits metrics by default in a format that is supported by Prometheus. Monitoring and observing those metrics is optional. Any software that can pull metrics that supports the same format could be used, but we only ensure it works with Prometheus and Grafana versions.
	- **Prometheus >= v2.0**
	- **Grafana >= v2.5**
- Sreaming (Optional, to be deprecated in future): Event streaming software as a dependency is only required when ElasticSearch is being used or when Temporal is deployed across multiple data centers, however in future releases of Temporal, third party event streaming software will likely cease to be needed as dependency for both. 
	- **Kafka v2.1.1 & Zookeeper v3.4.6**
```